### ISSUE TITLE
Node Explorer: UI freezes whilst attempting invalid login

### ISSUE THREAD

**Issue by joeldudleyr3** [corda/corda#3802]
The Node Explorer login windows freezes up when an invalid login is attempted, essentially blocking whilst waiting for a connection timeout.

This only means that the Node Explorer is blocked for 10 seconds whilst trying to connect to a wrong address/port.

Tracked in JIRA here: https://r3-cev.atlassian.net/browse/CORDA-1770.

**Comment by gitgabrio** [comment]
@joeldudleyr3  @mikehearn 
Hi. I just tried with releas/os/4.3 - started DemoBench and then a standalone NodeExplorer. Whenever I put wrong credentials (username/password/both) I've got an almost immediate "failure" popup (no freeze/lock). Should we close this ticket ?
I've created this one https://github.com/corda/corda/issues/5543 that is related to wrong connection address, and not with the login credentials.
